prompb/write/v2/types.proto

```diff
@@ -111,23 +113,43 @@ message Histogram {
   // Negative Buckets.
   repeated BucketSpan negative_spans =  8 [(gogoproto.nullable) = false];
   // Use either "negative_deltas" or "negative_counts", the former for
-  // regular histograms with integer counts, the latter for float
-  // histograms.
+  // regular exponential histograms with integer counts, the latter for exponential
+  // float histograms.
   repeated sint64 negative_deltas    =  9; // Count delta of each bucket compared to previous one (or to zero for 1st bucket).
   repeated double negative_counts    = 10; // Absolute count of each bucket.
 
   // Positive Buckets.
```

Comment:
```suggestion
  // Positive Buckets.
  // In case of the use of "custom_bounds" the positive buckets are interpreted as follows:
  // the span offset+length points to an the index of the custom_bound array or +Inf if indexing outside
  // the array. The counts and deltas have the same meaning as for exponential histograms.
```

Reply:
Or... I thought we ignore spans (why should we use them?)

Reply:
For code reuse and also because there are plenty of sparse classic histograms out there. For example when you application works smoothly , only the first 1-2 buckets would be used out of 10-40 possibly